src/constants/tokens.ts

```diff
@@ -229,6 +235,7 @@ export const IndexCoopInverseEthereumIndex: Token = {
   indexTypes: [IndexType.leverage],
   isDangerous: true,
   url: 'ieth1x',
+  borrowedAssetSymbol: 'weth',
```

Comment:
Need confirmation that this works, as ETH was requested but not returned from AAVE.

Reply:
Should be correct. Usually it's the case that the interactions with contracts happen in WETH. Check the `Details` for borrowing ETH here: https://app.aave.com/?marketName=proto_arbitrum_v3 (it will say WETH)